denysyefimov_task1/denysyefimov_houseprices.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from sklearn.metrics import mean_squared_error
+from sklearn import linear_model
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
+
+
+def one_hot_encode(x):
+    return 1 if x == 'Partial' else 0
+
+
+def encode1(x):
+ if (x == 4):
+        return 3
+ else:
+        return x
+
+
+if __name__ == '__main__':
+    # add data in file
+    train = pd.read_csv('C:/Users/Gyrocopter_UA/PycharmProjects/pythonProject/data/train.csv')

```

Comment:
Just for the future:
try not to include paths, IPs, personal details on any other personalized or secret data into github commits
This kind of data may unintentionally leak some secret information about a business if you working in a commercial project